editor/plugins/script_text_editor.cpp

```diff
@@ -1721,6 +1721,9 @@ void ScriptTextEditor::_enable_code_editor() {
 		color_picker->set_raw_mode(true);
 	}
 
+	int picker_shape = EDITOR_GET("interface/inspector/default_color_picker_shape");
```

Comment:
Think should not be configurable, I don't think there's a real use case for it.
You can simply hardcode it, it should be enough.

Reply:
Is it OK to use `(EnumType)EDITOR_GET("...").operator int()` for casting enum value ?

Reply:
Oh nevermind in fact. I'm dumb. 
Forget about my comment, I thought the picker shape was to configure the shape of the cursor in the wheel/circle/square, not the whole picker mode.

So yeah, the PR looks good to me then.